csrc/pybind.cpp

```diff
@@ -159,6 +162,11 @@ void sort_gaussian_torch(int num_rendered,
         (uint64_t*)gaussian_keys_sorted.contiguous().data_ptr<int64_t>(), (uint32_t*)gaussian_values_sorted.contiguous().data_ptr<int>());
 }
 
+size_t get_sort_buffer_size_torch(int num_rendered)
+{
+    return get_sort_buffer_size(num_rendered);
+}
+
```

Comment:
This wrapper function is necessary because otherwise pybind11 tries to add bindings for the new `cudaStream_t` argument, which it doesn't know how to do.